src/viam/services/vision/vision.py

```diff
@@ -117,18 +167,17 @@ async def get_object_point_clouds(
         picture obtained from the specified 3D camera (using the specified segmenter).
 
         To deserialize the returned information into a numpy array, use the Open3D library.
+
         ::
 
-            import numpy as np
-            import open3d as o3d
+        # Grab the 3D camera from the machine
+        cam1 = Camera.from_robot(robot, "cam1")
 
-            object_point_clouds = await vision.get_object_point_clouds(camera_name, segmenter_name)
+        # Grab the object segmenter you configured on your machine
+        my_segmenter = VisionClient.from_robot(robot, "my_segmenter")
 
-            # write the first object point cloud into a temporary file
-            with open("/tmp/pointcloud_data.pcd", "wb") as f:
-                f.write(object_point_clouds[0].point_cloud)
-            pcd = o3d.io.read_point_cloud("/tmp/pointcloud_data.pcd")
-            points = np.asarray(pcd.points)
+        # Get the objects from the camera output
+        objects = await my_segmenter.get_object_point_clouds(cam1)
```

Comment:
I think for this a combination of the samples might be a good idea since someone did spend the time doing a more elaborate example.
```suggestion
        import numpy as np
        import open3d as o3d

        # Grab the 3D camera from the machine
        cam1 = Camera.from_robot(robot, "cam1")
        # Grab the object segmenter you configured on your machine
        my_segmenter = VisionClient.from_robot(robot, "my_segmenter")
        # Get the objects from the camera output
        objects = await my_segmenter.get_object_point_clouds(cam1)
        # write the first object point cloud into a temporary file
        with open("/tmp/pointcloud_data.pcd", "wb") as f:
            f.write(objects[0].point_cloud)
        pcd = o3d.io.read_point_cloud("/tmp/pointcloud_data.pcd")
        points = np.asarray(pcd.points)
```